__tests__/app.test.js

```diff
@@ -44,3 +41,43 @@ describe("TASK 3.5 --- GET - /api", () => {
       });
   });
 });
+describe.only("TASK 4 --- GET - /api/articles/:article_id", () => {
```

Comment:
I know you've done a second commit removing this, be careful about the .only in your testing.